### PR TITLE
RHODS-4543 Improved GPU tutorial notebooks

### PR DESCRIPTION
**Pull request by laubai**
RHODS-4543 involves updating quickstarts in the RHODS dashboard that refer to this repository and these notebook files. This PR improves the tutorial notebook files in this repository to make it easier for users to follow along from the quickstarts to these notebooks and back again, and better integrate the experience of using RHODS dashboard and Jupyter to do work.

### CODE REVIEW

File: torch-use-gpu.ipynb [hunk 1]
```diff
@@ -240,26 +308,34 @@
...
     "test_loader = torch.utils.data.DataLoader(testset, batch_size=batch_size,\n",
     "                                         shuffle=False, num_workers=2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f77af8df",
+   "metadata": {},
+   "source": [
+    "Review the dictionary and sample content from the data set:"
```
Comment (laubai) [inline torch-use-gpu.ipynb]: It's not clear whether the data scientists would need to write this dictionary for themselves, or whether this is somewhere in the FashionMNIST data set. @cam-garrison do you or Audrey know the expectation here?

Comment (cam-garrison) [reply torch-use-gpu.ipynb]: @laubai, the dictionary is defined within the cell below, called `labels_map`. This cell could possibly be removed as the comment in the cell below is redundant. 

Also, could maybe be replaced with 'Run the following cell to see images from the dataset with their corresponding labels.' I think that could be more meaningful.

File: torch-test-model.ipynb [hunk 2]
```diff
@@ -67,14 +125,21 @@
...
    "cell_type": "code",
    "execution_count": null,
    "id": "e4c24aa3-0b70-4290-8e70-c1ef30775f76",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# putting the model on our GPU and loading in those trained weights.\n",
     "model = SimpleNet().to( device )\n",
     "model.load_state_dict( torch.load(\"mnist_fashion_SimpleNet.pt\") )"
    ]
```
Comment (laubai) [inline torch-test-model.ipynb:145]: @cam-garrison After this point is where there's info missing from the gdoc

Comment (laubai) [reply torch-test-model.ipynb:145]: Cam mentioned on the GDoc that we can ignore this, as there are enough code comments to get users through the process.

**Comment by laubai** [comment]
@cam-garrison Can I get you to review this? :)

**Review by cam-garrison — APPROVED** [review]
updated nbs look great!

**Comment by laubai** [comment]
@cam-garrison I can't merge so I'll leave that to you :)
